superbench/benchmarks/micro_benchmarks/cuda_nccl_bw_performance.py

```diff
@@ -128,7 +128,7 @@ def _preprocess(self):
     def _process_raw_result(self, cmd_idx, raw_output):    # noqa: C901
         """Function to parse raw results and save the summarized results.
 
-          self._result.add_raw_data() and self._result.add_result() need to be called to save the results.
+        self._result.add_raw_data() and self._result.add_result() need to be called to save the results.
```

Comment:
Why change microbenchmark code for runner support?

Reply:
will remove